object_tracking_with_gcp/Object Tracking Tutorial.ipynb

```diff
@@ -0,0 +1,381 @@
+    }
+   },
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![image.png](attachment:image.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from pathlib import Path\n",
+    "import csv\n",
+    "from tqdm import tqdm\n",
+    "from random import random"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "VIDEO_BUCKET = \"object_tracking_bucket\" # We will store videos here\n",
+    "ANNOTATIONS_BUCKET  = \"object_tracking_annotations\" # We will store annotations here\n",
+    "LOCATION=\"us-central1\" \n",
+    "\n",
+    "# The column names of the input csv-s map to these ids.\n",
+    "TEAM_1_LABELS = [f\"team_1_player_{i}\" for i in [1, 10, 11, 2, 3, 4, 5, 6, 7, 8, 9]]\n",
+    "TEAM_2_LABELS = [f\"team_2_player_{i}\" for i in [1, 10, 11, 2, 3, 4, 5, 6, 7, 8, 9]]\n",
+    "FULL_LABELS = [*TEAM_1_LABELS, *TEAM_2_LABELS, \"Ball\"]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def make_bbox(top_x, top_y, width, height):\n",
```

Comment:
Maybe we should add some more comments like: 'we prepare some converters for GCP format'